src/ngraph/op/topk.hpp

```diff
@@ -69,13 +69,13 @@ namespace ngraph
             /// \param index_element_type produce indices. Currently, only int64 or int32 are
             ///                           supported
             /// \param compute_max Compute top k max or top k min?
-            /// \param sort SortType for sorting results, default - NONE
+            /// \param sort SortType for sorting results, default - SORT_VALUES
             TopK(const Output<Node>& arg,
                  const Output<Node>& k,
                  size_t top_k_axis,
                  const element::Type& index_element_type,
                  bool compute_max = true,
-                 SortType sort = SortType::NONE);
+                 SortType sort = SortType::SORT_VALUES);
 
```

Comment:
Let's make all the attributes have no defaults (i.e. required).

Reply:
I don't think we should change anything here (we can delete `Changes API` if we roll this back), and if we do change it I think we should remove the default. The bridge will have to make an explicit selection anyway, because it has to handle the non-default case.

Reply:
The old implementation actually produced a sorted output by default even though NONE was selected and the unit tests depend on the sorted order. The new implementation produces a non-sorted result by default, so if we revert this back to NONE then we need to have `Changes API` because the op behaves differently than it used to.